insights/specs/default.py

```diff
@@ -495,6 +495,9 @@ def httpd_cmd(broker):
     ntpq_leap = simple_command("/usr/sbin/ntpq -c 'rv 0 leap'")
     ntptime = simple_command("/usr/sbin/ntptime")
     numa_cpus = glob_file("/sys/devices/system/node/node[0-9]*/cpulist")
+    numastat = simple_command("/bin/numastat")
+    numastat_memory = simple_command("/bin/numastat -m")
+    numactl_hw = simple_command("/bin/numactl --hardware")
```

Comment:
These specs will need to be approved before they can be collected.  Also you'll need to add corresponding `simple_file` specs to insights_archive.py for the data stored in an insights archive.  To find the correct path use the command `mangle {command_string}` and place the corresponding output in a spec in insights/specs/insights_archive.py.